fog/load_testing/src/bin/ingest.rs

```diff
@@ -268,6 +325,33 @@ fn load_test(ingest_server_binary: &Path, test_params: TestParams, logger: Logge
                 ledger_db
                     .append_block(block, block_contents, None)
                     .expect("Adding block failed");
+
+                // Add the timestamp information to watcher for this block index - note watcher
+                // does not allow signatures for block 0.
+                if block.index > 0 {
+                    for src_url in watcher.get_config_urls().unwrap().iter() {
+                        let mut block_signature =
+                            BlockSignature::from_block_and_keypair(&block, &signer)
+                                .expect("Could not create block signature from keypair");
+                        block_signature.set_signed_at(block.index);
+                        watcher
+                            .add_block_signature(
+                                src_url,
+                                block.index,
+                                block_signature,
+                                format!("00/{}", block.index),
+                            )
+                            .expect("Could not add block signature");
+                    }
+                }
+                // Sanity check that watcher is behaving correctly
+                assert_eq!(
+                    watcher
+                        .highest_common_block()
+                        .expect("Could not get highest common block"),
+                    block.index,
+                );
```

Comment:
similarly, the load test wasn't writing anything to the watcher, so we had to add this code to make it work